pkg/kudoctl/cmd/install/install.go

```diff
@@ -213,60 +212,43 @@ func verifySingleFramework(name, previous string, gc *github.Client, k2c *kudo.C
 			if helpers.AskForConfirmation() {
 				// If Instance is a dependency we need to make sure installSingleInstanceToCluster is aware of it.
 				// By having the previous string set we can make this distinction.
-				err := installSingleInstanceToCluster(name, previous, *content.Path, gc, k2c)
-				if err != nil {
+				if err := installSingleInstanceToCluster(name, previous, bundle.Instance, kc); err != nil {
 					return errors.Wrap(err, "installing single Instance")
 				}
 			}
 		} else {
-			err := installSingleInstanceToCluster(name, previous, *content.Path, gc, k2c)
-			if err != nil {
+			if err := installSingleInstanceToCluster(name, previous, bundle.Instance, kc); err != nil {
 				return errors.Wrap(err, "installing single Instance")
 			}
 		}
 
 	}
-
 	return nil
 }
 
 // Todo: needs testing
 // installSingleFrameworkToCluster installs a given Framework to the cluster
-func installSingleFrameworkToCluster(name, path string, gc *github.Client, k2c *kudo.Client) error {
-	frameworkYaml, err := gc.GetFrameworkYaml(name, path)
-	if err != nil {
-		return errors.Wrapf(err, "getting %s-framework.yaml", name)
-	}
-	_, err = k2c.InstallFrameworkObjToCluster(frameworkYaml)
-	if err != nil {
+func installSingleFrameworkToCluster(name string, f *v1alpha1.Framework, kc *kudo.Client) error {
+	if _, err := kc.InstallFrameworkObjToCluster(f); err != nil {
 		return errors.Wrapf(err, "installing %s-framework.yaml", name)
 	}
-	fmt.Printf("framework.%s/%s created\n", frameworkYaml.APIVersion, frameworkYaml.Name)
+	fmt.Printf("framework.%s/%s created\n", f.APIVersion, f.Name)
 	return nil
 }
 
 // Todo: needs testing
 // installSingleFrameworkVersionToCluster installs a given FrameworkVersion to the cluster
-func installSingleFrameworkVersionToCluster(name, path string, gc *github.Client, k2c *kudo.Client) error {
-	frameworkVersionYaml, err := gc.GetFrameworkVersionYaml(name, path)
-	if err != nil {
-		return errors.Wrapf(err, "getting %s-framework.yaml", name)
+func installSingleFrameworkVersionToCluster(name string, kc *kudo.Client, fv *v1alpha1.FrameworkVersion) error {
+	if _, err := kc.InstallFrameworkVersionObjToCluster(fv); err != nil {
+		return errors.Wrapf(err, "installing %s-frameworkversion.yaml", name)
 	}
-	_, err = k2c.InstallFrameworkVersionObjToCluster(frameworkVersionYaml)
-	if err != nil {
-		return errors.Wrapf(err, "installing %s-framework.yaml", name)
-	}
-	fmt.Printf("frameworkversion.%s/%s created\n", frameworkVersionYaml.APIVersion, frameworkVersionYaml.Name)
+	fmt.Printf("frameworkversion.%s/%s created\n", fv.APIVersion, fv.Name)
```

Comment:
Let's switch over to a logging package instead of using fmt. We're using the "log" package right now in the controllers. At some point we'll move this to another logger if needed.

Reply:
See the controllers for usage.